task-2/src/Pizza.ts

```diff
@@ -0,0 +1,25 @@
+import { Consumable } from "./Consumable";
+
+export class Pizza extends Consumable {
+    private numberOfSlices: number;
+    private slicesEaten: number;
+
+    constructor(name: string, value: number, weight: number, isSpoiled: boolean, numberOfSlices: number) {
+        super(name, value, weight, isSpoiled);
```

Comment:
From the task description, the name should not be dynamic from parameters, but a static `'pizza'`. Just like `Bow` and `Sword`